app/views/vocab_sheets/_vocab_sheet_item.html.haml

```diff
@@ -5,9 +5,9 @@
       = link_to sign_url(vocab_sheet_item.sign_id), { class: "drawing vocab_sheet_drawing drawing vocab_sheet_drawing__size--#{size}" } do
         = image_tag "signs/#{vocab_sheet_image_width}-#{vocab_sheet_image_height}/#{vocab_sheet_item.drawing}"
       = form_tag vocab_sheet_item_path(vocab_sheet_item), method: :put do
-        = text_area_tag "item[name]", vocab_sheet_item.name, class: 'item-name vocab-sheet__text-input h3', id: "item_#{vocab_sheet_item.id}_name"
+        = text_area_tag "item[name]", vocab_sheet_item.name, class: 'item-name vocab-sheet__text-input h3', id: "item_#{vocab_sheet_item.id}_name", maxLength:vocab_sheet_max_field_length
         = hidden_field_tag "item[#{vocab_sheet_item.id}][old_name]", vocab_sheet_item.name, class: 'old_name'
-        = text_area_tag "item[maori_name]", vocab_sheet_item.maori_name, class: 'item-maori-name vocab-sheet__text-input h3 maori_gloss', id: "item_#{vocab_sheet_item.id}_maori_name"
+        = text_area_tag "item[maori_name]", vocab_sheet_item.maori_name, class: 'item-maori-name vocab-sheet__text-input h3 maori-gloss', id: "item_#{vocab_sheet_item.id}_maori_name", maxLength:vocab_sheet_max_field_length
```

Comment:
can you make `maxLength` lowercase? This gets it working when i tested in firefox.